drivers/hud_spm_report/extensions/hud_reports/universe_member_extension.rb

```diff
@@ -19,6 +19,26 @@ module UniverseMemberExtension
         inverse_of: :hud_reports_universe_members,
         optional: true,
       )
+      belongs_to(
+        :enrollment,
+        -> do
+          where(HudReports::UniverseMember.arel_table[:universe_membership_type].eq('HudSpmReport::Fy2023::SpmEnrollment'))
+        end,
+        class_name: 'HudSpmReport::Fy2023::SpmEnrollment',
+        foreign_key: :universe_membership_id,
+        inverse_of: :hud_reports_universe_members,
+        optional: true,
+      )
+      belongs_to(
+        :return,
+        -> do
```

Comment:
Should these, maybe, include `spm_` in the name?  I worry we're injecting `belongs_to :enrollment` into UniverseMember, and other HUD reports may want to re-use similar namespace.